src/common/crypto_s2k.c

```diff
@@ -15,6 +15,7 @@
 #include "compat.h"
 #include "crypto.h"
 #include "crypto_digest.h"
+#include "crypto_dh.h"
```

Comment:
If we make the `crypto_hkdf.[ch]` module I suggested, we won't need this include.